ui/src/app/cluster-workflow-templates/components/cluster-workflow-template-summary-panel.tsx

```diff
@@ -35,12 +34,9 @@ export const ClusterWorkflowTemplateSummaryPanel = (props: Props) => {
                         kind='ClusterWorkflowTemplate'
                         title='Update Cluster Workflow Template'
                         value={props.template}
-                        onSubmit={(value: WorkflowTemplate) => {
-                            services.clusterWorkflowTemplate
-                                .update(value, props.template.metadata.name)
-                                .then(clusterWorkflowTemplate => props.onChange(clusterWorkflowTemplate))
-                                .catch(err => props.onError(err));
-                        }}
+                        onSubmit={(value: WorkflowTemplate) =>
```

Comment:
`onSubmit` now requires a promise which we display in the UI

Reply:
![Screenshot 2020-08-05 at 11 07 21](https://user-images.githubusercontent.com/1142830/89448032-d9fbee80-d70b-11ea-8541-0370e46856a1.png)